Cargo.toml

```diff
@@ -36,25 +36,26 @@ assert_fs = "1"
 async-trait = "0.1"
 axum = { version = "0.6", features = ["http2"] }
 bumpalo = "3"
-cairo-lang-compiler = "2.9.1"
-cairo-lang-defs = "2.9.1"
-cairo-lang-diagnostics = "2.9.1"
-cairo-lang-doc = "2.9.1"
-cairo-lang-filesystem = "2.9.1"
-cairo-lang-formatter = "2.9.1"
-cairo-lang-language-server = "2.9.1"
-cairo-lang-lowering = "2.9.1"
-cairo-lang-parser = "2.9.1"
-cairo-lang-runner = "2.9.1"
-cairo-lang-semantic = "2.9.1"
-cairo-lang-sierra = "2.9.1"
-cairo-lang-sierra-to-casm = "2.9.1"
-cairo-lang-starknet = "2.9.1"
-cairo-lang-starknet-classes = "2.9.1"
-cairo-lang-syntax = "2.9.1"
-cairo-lang-test-plugin = "2.9.1"
-cairo-lang-test-runner = "2.9.1"
-cairo-lang-utils = { version = "2.9.1", features = ["env_logger"] }
+cairo-lang-casm = { git = "https://github.com/starkware-libs/cairo", rev = "f30960c9393148e6fee7cb1333635fad3995a6a9" }
```

Comment:
Please use current Cairo main - even if it won't pass some tests (pls ignore/change asserts)

Reply:
Ok, I reverted it, but that does kind of partially nerfs our ability to use mappings to generate correct diagnostics.

> Please use current Cairo main

Can you also clarify whether you meant main branch or the latest tag? 😅

Reply:
main branch :D 

> Ok, I reverted it, but without https://github.com/starkware-libs/cairo/pull/6717 we lose our ability to use mappings to generate correct diagnostics:

Yes, that's true :( 
We will update tests when we merge the needed changes on Cairo side